e2e/testcases/correction-birth/correct-birth-record-5.spec.ts

```diff
@@ -259,17 +264,14 @@ test.describe.serial(' Correct record - 5', () => {
 
       await expect(
         page.locator('#child-content #Date').getByRole('deletion')
-      ).toHaveText(
-        format(parseISO(declaration.child.birthDate), 'dd MMMM yyyy'),
-        { ignoreCase: true }
-      )
+      ).toHaveText(formatDateTo_ddMMMMyyyy(declaration.child.birthDate), {
+        ignoreCase: true
```

Comment:
Is the case something we need to ignore? Would be nice if the formatting was all we needed